ballerina/build.gradle

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in [54fcb0d](https://github.com/ballerina-platform/module-ballerina-persist/pull/1/commits/54fcb0def82f78c0e312798b888dadd74e69b77d)